comeet/UI/RoomsList/RoomsListViewModel.swift

```diff
@@ -64,6 +64,7 @@ class RoomsListViewModel : BaseViewModel {
         }
         fetcher.getRooms(organization: authenticator.getOrganization(), roomlist: email, start: startDateForRequest(), end: endDateForRequest()) { [weak self] (rooms, error) in
             guard error == nil else {
+                // TODO: surface error in UI?
```

Comment:
I'm adding https://cocoapods.org/pods/ALLoadingView to show loading indicators to the user while fetching and booking rooms. Maybe we could use it or a toast like component: https://cocoapods.org/pods/ToastSwift